src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -16,13 +19,52 @@ class Thurloe extends Actor with MyService {
   // this actor only runs our route, but you could add
   // other things here, like request stream processing
   // or timeout handling
-  def receive = runRoute(myRoute)
+  def receive = runRoute(thurloeRoutes)
 }
 
 
 // this trait defines our service behavior independently from the service actor
+@Api(value="/thurloe", description = "Thurloe service", produces = "application/json", position = 1)
 trait MyService extends HttpService {
 
+  import spray.httpx.SprayJsonSupport.sprayJsonUnmarshaller
+
+  def thurloeRoutes: Route =
+  pathPrefix("thurloe") {
+    path(Segment / "get" / Segment) { (userId, key) =>
```

Comment:
If/when we finish [spray-swagger](https://github.com/gettyimages/spray-swagger)'ing these routes (it looks like the data models have annotations, but not the routes), `def thurloeRoutes = pathPrefix...` will need to be refactored. The spray-swagger annotations need to be on [individual](https://github.com/mhamrah/spray-swagger-sample/blob/master/src/main/scala/PetHttpService.scala) `def` methods. Then the overall route will be composed of calling into the individual routes.

Also, I believe unless the route takes parameters, the convention is to use `val` instead of `def` for the composed `thurloeRoutes` route.

```scala
// in spray
val myRouteNoParam: Route
def myRouteWithParam(id: Int): Route
// in case classes / objects
case object MySingletonNoParam
case class MyClassWithParam(other: String)
```